CHANGELOG.md

```diff
@@ -1108,13 +1108,13 @@ return (
 
 - Bump all packages to next major for Chakra UI version 2.
 - Omit `src` directory from being published to npm
-- Removed the `isFullWidth` deprecation. Use `width=full` or `width=100%`
+- Removed the `isFullWidth` deprecation. Use `width=full` or `width='100%'`
```

Comment:
If you're going to update the `width='100%'`, you should consider updating `width=full` as well

Reply:
Very fair point ! Authoring this while in the context of actually migrating wasn't the most optimal haha.
Done ✅